packages/admin-ui/src/form-control/form-control-message.tsx

```diff
@@ -22,7 +22,7 @@ export function FormControlMessage(props: FormControlMessageProps) {
         </Text>
       ) : null}
       {hasError ? (
-        <Text variant="detail" tone="critical">
+        <Text variant="detail" tone="critical" role="alert">
```

Comment:
Critical messages [must have the `alert` role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/alert_role).

Reply:
Is that linked in the docs?

Reply:
good catch!

Reply:
@kevinch, we can document it on the a11y section once the `form-control` documentation is available.